glaring-client/src/components/Sidebar/Index.jsx

```diff
@@ -26,13 +28,13 @@ const Header = styled.div`
 `;
 
 export default function Sidebar(props) {
-  const { groups, setCurrentGroup } = props;
+  const { groups, setCurrentGroup, cancel } = props;
 
   console.log("PROPS", props)
 
   const groupList = groups.map((element) => {
     const { color, name, id } = element;
-    return <NavGroup key={id} color={color} name={name}  setCurrentGroup = {setCurrentGroup} />;
+    return <NavGroup key={id} id={id} color={color} name={name} setCurrentGroup = {setCurrentGroup} cancel={() => cancel(id)} />;
```

Comment:
`cancel={(event) => cancel()}` is safer, we don't want to risk destroying data on pageload.